tokens/mainnet.json

```diff
@@ -32,6 +32,14 @@
       "decimals": 0,
       "description": "Montycoin is an incentivization token made for fun and immersion into MontaiL's DJ'ing career. With widely growing uses.",
       "logoURI": "https://raw.githubusercontent.com/alephium/tokens-meta/master/logos/TAIL.png"
+    },
+    {
+      "id": "1a281053ba8601a658368594da034c2e99a0fb951b86498d05e76aedfe666800",
+      "name": "Ayin",
```

Comment:
We should keep it consistent with the on-chain contract, so the name here should also be `AYIN`.

Reply:
I changed the the token name to be `Ayin` 👌